pkgs/data/fonts/nahid-fonts/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "nahid-fonts";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "rastikerdar";
+    repo = "nahid-font";
+    rev = "v${version}";
```

Comment:
Use `pname` to define the `repo` attribute.

Reply:
`pname` is
`"nahid-fonts"`
but `repo` is
`"nahid-font"`